test/core/utils/flattened-tree.js

```diff
@@ -86,11 +89,20 @@ describe('axe.utils.getFlattenedTree', function() {
     assert(tree[0].parent === null);
   });
 
+  it('creates virtual nodes in the correct order', function() {
+    fixture.innerHTML = '<p><b><i></i></b><p><u><s></s></u>';
```

Comment:
The code created an extra `p` element instead of closing it so the resulting asserts may fail when updating this.

```suggestion
    fixture.innerHTML = '<p><b><i></i></b></p><u><s></s></u>';
```